commands/mentor.py

```diff
@@ -0,0 +1,28 @@
+import requests
+import random
+from disco.bot import Bot,Plugin
+from disco.types.message import MessageEmbed
+from config_mock import AnnounceBotConfig, TestPluginConfig
+
+MENTOR_ID = 487807730506006540
+
+@Plugin.with_config(TestPluginConfig)
+class MentorPlugin(Plugin):
+
+    def get_avail_mentors(self):
+       return [u.user.id for u in self.bot.client.state.guilds.get(487611837664198679).members.values() if MENTOR_ID in u.roles and u.user.presence.status != "OFFLINE"]
+
+    def ping_mentor(self, mentor, author, content):
+        self.bot.client.state.channels.get(self.config.channel_IDs.get('MENTOR_CHANNEL')).send_message(self.config.messages.get("help_message").format(mentor, author, content))
+
+    def concat_name_discrim(self, name, discrim):
+        return "{}#{}".format(name, discrim)
+
+    @Plugin.command("helpme", "<content:str...>")
+    def on_help_command(self, event, content):
+        if event.channel.is_dm:
+            mentors_available = self.get_avail_mentors()
+            if mentors_available:
+                self.ping_mentor(mentors_available[random.randint(0, len(mentors_available) - 1)], self.concat_name_discrim(event.msg.author.username, event.msg.author.discriminator), content)
+            else:
+                event.msg.reply(self.config.messages.get("no_mentors_available"))
```

Comment:
can we maybe just put the message in the mentors channel without ping anyways? and just notify the user waiting times may be a little longer?

Reply:
I was always envisioning a DM at this step, but I guess a channel works as well. Is there a benefit to doing it through a channel?

Reply:
i'm fine with removing the ping, since it feels very spammy otherwise. if we put the the message in the channel and one of the mentors picks it up, how do you guys think we should log that?

Reply:
i think the ping itself could be nice (maybe try it and see what people think)? it's mostly a thing of not dropping requests when nobody is available as it basically means people have to wait and try again later. still logging it without ping means if some is around 5m later they can take it